### PR TITLE
React component version

### PR DESCRIPTION
**Pull request by ramasilveyra**
New Auth0 web header

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -1,129 +0,0 @@
-var pkg = require('./package');
```
Comment (alejofernandez) [inline Gruntfile.js]: Goodbye grunt! Hello webpack ;-)

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (alejofernandez) [inline LICENSE.md]: I believe we have a custom license file somewhere, you can ask crew-brokkr

File: component.json [hunk 3]
```diff
@@ -1,29 +0,0 @@
-{
```
Comment (alejofernandez) [inline component.json]: Goodbye component 😉

**Comment by alejofernandez** [comment]
Looks good so far 🙌

File: Gruntfile.js (hunk 1 above)
Comment (ramasilveyra) [inline Gruntfile.js]: Maybe I will use gulp to trigger the webpack builds and because we need a task to upload to our CDN.

**Pull request merged**